docs/02-guides/override-a-component.mdx

```diff
@@ -2,14 +2,177 @@
-  customize the default theme
+  customize the chocolatine theme
 sidebar_position: 1
 ---
 
 <p>{frontMatter.description}</p>
 
-:::info WIP
+## Understanding theme overrides
 
-See https://developers.front-commerce.com/docs/2.x/essentials/extend-the-theme
+Front-Commerce web application is a fully featured e-commerce
+[universal](https://cdb.reacttraining.com/universal-javascript-4761051b7ae9)
+React application aimed at providing a sane base for your own theme. Even though
+you could reuse pages and components individually in a totally different theme,
+most of the time you might find easier and faster to start with the base theme
+and iterate from there.
+
+**Theme override is what allows you to:**
+
+- customize the
+  [chocolatine theme](/docs/remixed/extensions/theme-chocolatine/),
+- customize how an extension is displayed,
+- upgrade Front-Commerce and benefit from the latest component improvements,
+- or create a slightly different theme for the Black Friday in confidence!
+
+Having an understanding of
+[themes in Magento](https://developer.adobe.com/commerce/frontend-core/guide/templates/),
+[child themes in Wordpress](https://developer.wordpress.org/themes/advanced-topics/child-themes/)
+/
+[Prestashop](https://devdocs.prestashop-project.org/8/themes/reference/template-inheritance/parent-child-feature/),
+[templates override in Drupal](https://www.drupal.org/docs/develop/theming-drupal/twig-in-drupal/working-with-twig-templates),
+[themes in CakePHP](https://book.cakephp.org/3.0/en/views/themes.html) will help
+you understand Front-Commerce’s theme overrides because they all share the same
+philosophy. If you have no previous experience with these other implementations,
+let’s see how it works!
+
+Your own theme will be located in its own folder and will use default components
+from parent theme(s) — at least from Front-Commerce chocolatine theme. You would
+then copy the files you want to override in your theme folder by maintaining an
+identical file structure. Your component will then be used instead of the one in
+your parent theme(s).
+
+This translates in those three steps:
+
+1. configure your custom theme and use it in your application
+2. copy the file (`jsx`, `scss` or `gql`) you want to override in the `theme`
+   folder of the `app` directory
+3. customize its content in your extension directory
+
+## Creating the `theme` folder
+
+### Using the skeleton as base
+
+First we need to create the `theme` directory in your `app` folder
+
+```
+my-project
+└── app
+    └── theme
+```
+
+That’s it! You have successfully created the folder that will be the root of
+your custom theme !
+
+### In your own extension
+
+You can also extend theme in your extension, please read
+[Register an extension](/docs/remixed/guides/register-an-extension) guide to
+know how to create your own extension.
+
+First in your extension definition file, add the `theme` entry :
+
+```javascript title="extension/my-extension/index.ts"
```

Comment:
Easier to read if you assign it a "fake" name.

```suggestion
```javascript title="extension/theme-acme/index.ts"
```